js/src/structured-data-blocks/how-to/components/HowTo.js

```diff
@@ -306,28 +306,50 @@ export default class HowTo extends Component {
 
 		return (
 			<div className="schema-how-to-duration">
-				<span>{ __( "Time needed:", "wordpress-seo" ) }&nbsp;</span>
-				<input
-					className="schema-how-to-duration-input"
-					type="number"
-					value={ attributes.hours }
-					onFocus={ () => this.setFocus( "hours" ) }
-					onChange={ ( event ) => setAttributes( { hours: Math.max( 0, event.target.value ) } ) }
-					placeholder="HH"/>
-				<span>:</span>
-				<input
-					className="schema-how-to-duration-input"
-					type="number"
-					value={ attributes.minutes }
-					onFocus={ () => this.setFocus( "minutes" ) }
-					onChange={ ( event ) => setAttributes( { minutes: Math.min( Math.max( 0, event.target.value ), 60 ) } ) }
-					placeholder="MM" />
-				<IconButton
-					className="schema-how-to-duration-button editor-inserter__toggle"
-					icon="trash"
-					label={ __( "Delete total time", "wordpress-seo" ) }
-					onClick={ () => setAttributes( { hasDuration: false } ) }
-				/>
+				<fieldset>
+					<legend
+						className="schema-how-to-duration-legend"
+					>
+						{ __( "Time needed:", "wordpress-seo" ) }&nbsp;
+					</legend>
+					<label
```

Comment:
We have a lot of duplicate code for labels and inputs now, maybe we can move this to a separate function?